systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -63,7 +63,7 @@ public static void waitForPodsReady(LabelSelector selector, int expectPods, bool
         int[] counter = {0};
 
         TestUtils.waitFor("All pods matching " + selector + "to be ready",
-            Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Duration.ofMinutes(6).toMillis(),
+            Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, ResourceOperation.rollingUpdateTimeout(expectPods),
```

Comment:
Maybe here will be conflict with #3278 -> but I will resolve it in my PR

Reply:
Ye, if it will be merged sooner, I will resolve conflicts. Looks you have some failures in tests there.